### PR TITLE
optimize: add configuration center upload configuration interactive script (apollo,consul,zk)

### PR DESCRIPTION
**Pull request by imherewait**
…(apollo,consul,zk) #4332Realize configuration center upload configuration interactive script (apollo,consul,zk) #4332

- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
Realize configuration center upload configuration interactive script (apollo,consul,zk) #4332

### Ⅱ. Does this pull request fix one issue?
#4332

### CODE REVIEW

**Comment by slievrly** [comment]
@imherewait Please help on the  https://github.com/seata/seata/blob/develop/script/config-center/README.md introduce the use of interactive script.

---
请帮忙在https://github.com/seata/seata/blob/develop/script/config-center/README.md 介绍下可交互脚本的使用。

**Comment by imherewait** [comment]
> @imherewait Please help on the https://github.com/seata/seata/blob/develop/script/config-center/README.md introduce the use of interactive script.
> 
> 请帮忙在https://github.com/seata/seata/blob/develop/script/config-center/README.md 介绍下可交互脚本的使用。

好的

**Comment by imherewait** [comment]
> @imherewait Please help on the https://github.com/seata/seata/blob/develop/script/config-center/README.md introduce the use of interactive script.
> 
> 请帮忙在https://github.com/seata/seata/blob/develop/script/config-center/README.md 介绍下可交互脚本的使用。

已完成，请帮忙再review一下

File: script/config-center/README.md [hunk 1]
```diff
@@ -14,12 +14,72 @@ you only need to follow the instructions below and keep the corresponding config
 | store.db.user            | service.vgroupMapping.default_tx_group                       |
 | store.db.password        | service.disableGlobalTransaction                             |
 
+## Script Introduction
+
+The Script has interactive and non-interactive configuration modes,different patterns are distinguished by different file names.
+
+interactive mode(*-config.sh or *-config.py):  the script starts the config program in interactive mode on the command line, prompting you for each option.
```
Comment (slievrly) [inline script/config-center/README.md:21]: Is it reversed?

Comment (imherewait) [reply script/config-center/README.md]: > Is it reversed?

已修复

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**